src/modules/services_manager_target.rb

```diff
@@ -139,8 +140,15 @@ def inspect
 
     def save
       return true unless modified?
+
       log.info('Saving default target...')
-      Yast2::Systemd::Target.set_default(default_target)
+      unless Yast2::Systemd::Target.find(self.default_target)
```

Comment:
NP: matter of taste, I guess. I'd extract this to its own method, something like 

```ruby
def save
  return true unless modified?
  
  ensure_target

  Yast2::Systemd::Target.set_default(self.default_target)
end

def ensure_target
  return if Yast2::Systemd::Target.find(self.default_target)

  log.info("#{self.default_target} not found, using mode fallback instead")
  Report.Warning(
    _("...")
  )
end